variables.js

```diff
@@ -0,0 +1,17 @@
+function variables() {
+
+    let userinput;
+    userinput = document.getElementById("user").value;
+
+    let passinput;
+    passinput = document.getElementById("pass").value;
+
+    let GFG_Var;
+ 
+    userdisplay = (userinput);
+    document.getElementById('result').innerHTML = `${userdisplay}` ;
```

Comment:
What does surrounding the `userDisplay` variable with quotes accomplish?

Reply:
not really sure i just saw this syntax on a website. do i not need it?